lib/ret_web/channels/hub_channel.ex

```diff
@@ -886,6 +921,18 @@ defmodule RetWeb.HubChannel do
        ),
        do: deny_join()
 
+  defp join_with_hub(
```

Comment:
This pattern match is the actual thing enforcing the invite permissions right? And I assume the `account_can_update` check is so that if you have permission to update the hub you bypass the invite check altogether?

Reply:
Correct.